ra/ra_test.go

```diff
@@ -22,6 +22,7 @@ import (
 
 	"github.com/jmhodges/clock"
 	"github.com/letsencrypt/boulder/bdns"
+	caPB "github.com/letsencrypt/boulder/ca/proto"
```

Comment:
General style is all lowercase, so `capb`.

Reply:
Fixed in d1f6fe7 - thanks!